Dockerfile

```diff
@@ -16,6 +16,5 @@ RUN cd /usr/bin && \
     microdnf install -y gzip qemu-img libguestfs-tools-c && \
     microdnf clean all -y
 COPY --from=builder /app/kubevirt-disk-uploader /usr/local/bin/kubevirt-disk-uploader
-COPY run-uploader.sh /usr/local/bin/run-uploader.sh
 
-ENTRYPOINT ["/usr/local/bin/run-uploader.sh"]
+ENTRYPOINT ["/usr/local/bin/kubevirt-disk-uploader"]
```

Comment:
On line 12 it would be nice to have it automated, because e.g. currently version 1.1.1 is available.

Reply:
I see, but I want to get rid of `virtctl` at all and use VMExport object directly, then I think this automation won't be required.